src/strings.ts

```diff
@@ -137,6 +137,15 @@ export const FiscalCode = PatternString(
 
 export type FiscalCode = t.TypeOf<typeof FiscalCode>;
 
+/**
+ * A fake Fiscal Number (for persons)
```

Comment:
```suggestion
 * A sandbox Fiscal Code (used for development)
 * Note that this pattern doesn't generate valid fiscal codes
```

Reply:
@gunzip @francescopersico could you check that the pattern doesn't indeed generate valid fiscal codes?

Reply:
```
/**
 * Generate a fake fiscal code.
 * Avoids collisions with real ones as we use
 * a literal "Y" for the location field.
 */
```

Reply:
@cloudify The generated fiscalcode IS valid, is just associated to a (at the moment) not existing "Codice catastale"